src/controllers/node/NodeTransactionPoolController.ts

```diff
@@ -29,12 +30,14 @@ export default class NodeTransactionPoolController extends AbstractController<No
 	 * @param res Express Response
```

Comment:
Just a little note to adjust the comment describing the function params.

Reply:
This comment was supposed to reference line 28

Reply:
Good catch.